### DISCUSSION TITLE
Elementor 3.6.4 Update Broke my website

### DISCUSSION THREAD

**Discussion by nardij73** [elementor/elementor#18446] [Developers Community]
Hi all, I'm hoping some one can help me. Every time I try to update elementor to 3.6.4 it takes my site down and all I get is the white screen of death. The last version that worked is 3.5.6.

Does anyone have any ideas how to update without taking my site down?

**Comment by shilo-ey** [comment]
Hi @nardij73 ,

Thank you for reaching out.

In Elementor v3.6 there were some code deprecations that were planned since v2.8. This made some 3rd party plugins that are not well maintained to conflict and therefore to cause breaks in your site.

Please make sure all of your third party plugins are compatible and updated.

This should resolve the issue you encounter

**Reply by nardij73** [reply]
I deleted almost all plugins and updating elementor to the newest version gave me the white screen of death again

**Reply by shilo-ey** [reply]
Thanks.
 
It seems like you have PLUGIN/THEME conflict somewhere in your install.

Start by deactivating all other add-ons and plugins other than Elementor Core & Elementor Pro (if installed) - then proceed to edit and see if the issue has been resolved. You may need to switch to the default theme to rule out your theme as the cause of the conflict.

If the issue is resolved by following the above action the proceed to reactivate each plugin 1 at a time and checking the editor till you find the one causing the conflict.

If plugin and theme deactivation didn't work - please try to deactivate all Experiments and activate them one by one until you identify the buggy experiment (Elementor > Settings > Experiments tab)

If the above process does not resolve the issue then check on Memory Limit set for your site together with PHP version - the easiest way to do that is by going to Elementor > System Info & scanning through it. Would also help if you post that info here for further assistance.

If you have questions or need general support, Please try:

* [Elementor Community](https://www.facebook.com/groups/Elementors/) on Facebook
* [Elementor Knowledge base](https://docs.elementor.com)

If you have purchased [Elementor Pro](https://elementor.com/pro), you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.


thanks.

**Reply by nardij73** [reply]
Thank you for this. Sorry I'm new to web-building any help would be greatly appreciated.

I deleted all my plugins except my theme plugin and I get the white screen of death. 
This is the error that I get on said white screen. 

Fatal error: Uncaught Error: Class 'Elementor\Scheme_Color' not found in /var/www/wp-content/plugins/fotofly-core/shortcode/widgets/fotofly-hero-header.php:216 Stack trace: #0 /var/www/wp-content/plugins/elementor/includes/base/controls-stack.php(2280): Fotofly\Widgets\Fotofly_Hero_Header->_register_controls() #1 /var/www/wp-content/plugins/elementor/includes/base/controls-stack.php(494): Elementor\Controls_Stack->init_controls() #2 /var/www/wp-content/plugins/elementor/includes/base/widget-base.php(177): Elementor\Controls_Stack->get_stack() #3 /var/www/wp-content/plugins/elementor/includes/base/controls-stack.php(300): Elementor\Widget_Base->get_stack() #4 /var/www/wp-content/plugins/elementor/includes/base/controls-stack.php(2369): Elementor\Controls_Stack->get_controls() #5 /var/www/wp-content/plugins/elementor/includes/base/controls-stack.php(1105): Elementor\Controls_Stack->sanitize_settings(Array) #6 /var/www/wp-content/plugins/elementor/includes/base/controls-stack.php(2116): Elementor\Controls_Stack->get_data('se in /var/www/wp-content/plugins/fotofly-core/shortcode/widgets/fotofly-hero-header.php on line 216

**Comment by aaliyahfraser** [comment]
I have also posted a blog on this topic. This is because Elementor has so many free amazing add-ons. Easy-to-use features made creating websites a breeze. Here we talk about “Best Elementor Addons and Plugins for WordPress 2023 – Compared” You can check out it at – https://quomodosoft.com/best-elementor-addons-and-plugins/
